flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectQueryITCase.java

```diff
@@ -1062,6 +1062,87 @@ public void testSumAggWithGroupKey() throws Exception {
         tableEnv.executeSql("drop table test_sum_group");
     }
 
+    @Test
+    public void testMinAggFunctionPlan() {
```

Comment:
it's better the plan test should not be in ITCase

Reply:
Current Hive ITCase covered all the plan related tests, we can improve this by another separate PR